integreat_cms/cms/templates/statistics/statistics_sidebar.html

```diff
@@ -50,3 +50,32 @@ <h3 class="heading font-bold text-black">
         </button>
     </form>
 </div>
+<div class="rounded border border-solid border-blue-500 shadow-2xl bg-white mb-2">
```

Comment:
Do you think it makes sense to extract this box and the box above into separate files, or does it not make sense (at the moment)? :)